src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
```suggestion
        Machine truck = new Truck();
```
для того щоб використати поліморфізм зазвичай використовуються абстракні типи для посиланян (інтерфейс абстрактний клас) і конкретні типи як об'єкт. Аналогічно до бульдозера і екскаватора